rbac/internal/views.py

```diff
@@ -504,6 +509,52 @@ class SentryDiagnosticError(Exception):
     pass
 
 
+def migration_resources(request):
```

Comment:
As we chatted about, we'll probably want to kick this off in a worker process so that if the deletes exceed the http timeout, it won't be an issue.